app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -64,10 +64,11 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         });
 
     // Try pulling data from GraphQL, if not fallback to json from cornell dining
-    NetworkUtilities.getEateries(presenter,this);
+    NetworkUtilities.getEateries(this);
     if (JSON_FALLBACK) {
       new ProcessJson().execute("");
     }
+    NetworkUtilities.getCtEateries(this);
```

Comment:
What happens when network utilities for ct eateries fail? is there a catch somewhere or a 404 page for the ui?

Reply:
i was gona show a toast in future pr for ui that ct is not available and keep showing campus eateries

Reply:
sounds good!